core/includes/icon.inc

```diff
@@ -294,8 +295,23 @@ function theme_icon(array $variables) {
     if (strpos($svg_contents, '<svg') === 0) {
       // Clean out any embedded XSS within the SVG. This very-restrictive set
       // of options should be adequate for icons.
-      $svg_contents = filter_xss($svg_contents, array('svg', 'use', 'title',
-        'desc', 'defs', 'linearGradient', 'stop', 'rect', 'circle', 'path'));
+      // @todo: Allow <line> for now for the purpose of layout template preview
+      // icons, but follow up properly in
+      // https://github.com/backdrop/backdrop-issues/issues/6576
```

Comment:
@jenlampton same as this...

Reply:
This is now in core, with #4796 merged :) ...I've removed it from this PR with this commit: [`889f1a7` (#4769)](https://github.com/backdrop/backdrop/pull/4769/commits/889f1a7a738fefaf0e27bb9f5e88e61cbbee88cb)